docs/getting_started.md

```diff
@@ -132,36 +132,35 @@ Experiment with the graph range parameters and other settings.
 Let us make this more interesting and start some example targets for Prometheus
 to scrape.
 
-The Go client library includes an example which exports fictional RPC latencies
-for three services with different latency distributions.
+The next steps are for `node_exporter` which is targeted to *nix type OS, but for Windows
+[`wmi_exporter`](https://github.com/martinlindhe/wmi_exporter)
 
-Ensure you have the [Go compiler installed](https://golang.org/doc/install) and
-have a [working Go build environment](https://golang.org/doc/code.html) (with
-correct `GOPATH`) set up.
-
-Download the Go client library for Prometheus and run three of these example
-processes:
+Start by downloading the latest version of node_exporter from the
+[Github Releases](https://github.com/prometheus/node_exporter/releases)
+* Look under assets and find the tar.gz file that most closely matches your operatings system.
+** If you don't know it's likely `amd64` for 64-bit and either `linux` or `darwin` (macOS).
 
 ```bash
-# Fetch the client library code and compile example.
-git clone https://github.com/prometheus/client_golang.git
-cd client_golang/examples/random
-go get -d
-go build
+# Go to the location of the download.
+cd ~/Downloads
```

Comment:
You're making a presumption that a browser was used and that this is the download location, it's easier to not mention this and presume that the user knows where their files are.